doc/tutorials/tutorials.txt

```diff
@@ -55,7 +55,6 @@ mlpack.
 
  - \ref bindings
  - \ref cv
- - \ref hpt
```

Comment:
It had an issue with this, and I don't know why.

Reply:
I wonder if it's because `hpt` is also a namespace.  Would you mind trying changing the name of the `hpt` `@page` in `doc/guide/hpt.hpp` and then adapting the name here and re-adding it?  I think that will work.

Reply:
Happy to help work this one out; just let me know if you want me to take a whack at it.  But this tutorials list is the only reference to the hyperparameter tuning guide, so we should make sure that we add it back in (in one form or another).  I unresolved the conversation so it doesn't get lost in the huge PR :)

Reply:
Ah, sorry for prematurely resolving; added in 8bc4d69